src/test/java/io/weaviate/integration/client/schema/ClusterSchemaTest.java

```diff
@@ -494,4 +448,40 @@ public void shouldBatchObjectsWithNestedProperties_NoSchema2() {
     schemaClass = utils.getClass(client2, className);
     utils.assertThatSchemaPropertiesHaveDataTypes(expectedProps, schemaClass);
   }
+
+  private void validateClassReplicationSettings(int replicationFactor, boolean asyncEnabled) {
```

Comment:
(nitpicking) maybe `assertClassReplicationSettings` to indicate asserts are called inside?

Reply:
will change the name, thanks!